modules/async-icq/testing/simapp/export.go

```diff
@@ -108,12 +115,22 @@ func (app *SimApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []
 	// reinitialize all validators
 	app.StakingKeeper.IterateValidators(ctx, func(_ int64, val stakingtypes.ValidatorI) (stop bool) {
 		// donate any unwithdrawn outstanding reward fraction tokens to the community pool
-		scraps := app.DistrKeeper.GetValidatorOutstandingRewardsCoins(ctx, val.GetOperator())
-		feePool := app.DistrKeeper.GetFeePool(ctx)
+		valAddr, _ := sdk.ValAddressFromBech32(val.GetOperator())
```

Comment:
nit: We should check the err here and ujse the new `StringToBytes` per the SDK. Same for other usage of `sdk.ValAddressFromBech32`
```go
valBz, err := app.StakingKeeper.ValidatorAddressCodec().StringToBytes(val.GetOperator())
if err != nil {
	panic(err)
}
```